templates/template.html

```diff
@@ -8,15 +8,15 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="theme-color" content="#8338EC"/>
 
-  <title>Warwick's Awesome Speedruns and Demos 2021</title>
+  <title>Warwick's Awesome Speedruns and Demos Summer</title>
```

Comment:
MD: We should indicate what year WASD Summer is for if it's going to be a regular event like the main one in February.

Reply:
Thanks! Should now all be up to date :)